docs/examples/incl/ArgParser.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef ARGPARSER_HPP
+#define ARGPARSER_HPP
+
+#include <boost/program_options/variables_map.hpp>
```

Comment:
Simplify this example code and don't use the boost library. The example file should be a C example and it should contain at least two bugs: one which is found by a Clang Static Analyzer (for example division by zero) and one which is found by the Clang Tidy analyzer.

Reply:
Done. C example is in the repo.